hamp_radar/iquick.py

```diff
@@ -12,6 +12,34 @@
 from decoders import get_decoders, decode_ppar, decode_time
 
 
+def get_tag_size(data):
+    """
+    Extracts the tag (also known as signature) and the pointer for size
+    from 'data' assuming layout as in Meteorological Ka-Band Cloud Radar
+    MIRA35 Manual, section 2.3.1 'Definition of chunk common structure'
+
+    The tag is the first 4 bytes of data and the pointer for size is the
+    next 4 bytes. Bytes for tag are decoded to ascii string unless no valid
+    tag can be formed (see below). Bytes for size are interpreted as an integer
+
+    If the first 4 bytes cannot be decoded as an ASCII string, (e.g. because a
+    byte in the data for the tag represents a uint not in range 0-255), then
+    UnicodeDecodeError is swallowed and tag=None and size=0 are returned.
```

Comment:
I'd probaby just describe the externally visible behaviour in the docstring, as that's what users will use and the docstring isn't super useful for interpreting the internals of the function, as one could also just read the function body or have comments within the function.

```suggestion
    If the first 4 bytes cannot be decoded as an ASCII string, tag=None and size=0 are returned.
```